administrator/components/com_templates/Model/TemplateModel.php

```diff
@@ -233,6 +232,27 @@ public function prepareCoreFiles($dir, $client, $element, $template)
 		return;
 	}
 
+	/**
+	 * Method to get the state of the Installer Override Plugin.
+	 *
+	 * @return  array  Array of relevant plugins and whether they are enabled or not.
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getPluginState()
+	{
+		$db = $this->getDbo();
```

Comment:
Ca we not use here PluginHelper::getPlugin?

Reply:
I think you can use Joomla\CMS\Plugin\PluginHelper;

Please try in file /administrator/components/com_templates/View/Template/HtmlView.php in line 147 this:
$this->pluginState = PluginHelper::isEnabled('installer', 'override');

Do not forget 
use Joomla\CMS\Plugin\PluginHelper; 
in the head.

Reply:
@laoneo @astridx Thanks :)